invenio.cfg

```diff
@@ -329,3 +330,7 @@ CDS_EOS_OFFLOAD_ENABLED = False
 CDS_LOCAL_OFFLOAD_ENABLED = False
 CDS_LOCAL_OFFLOAD_FILES = ["file.txt", "file2.txt"]
 CDS_LOCAL_OFFLOAD_STORAGE = ""
+
+# CDS Administration panel
+# ========================
+USERS_RESOURCES_ADMINISTRATION_ENABLED = True
```

Comment:
I think we really need to change `invenio-users-resources` and now enable it by default. Feel free to add a commit there